cypress.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+	e2e: {
+		setupNodeEvents(on, config) {
+			// implement node event listeners here
+		},
+		baseUrl: 'http://localhost:3000',
```

Comment:
It will work for now, but in the future we would like to read that information from environment variables.